### PR TITLE
Move test files to a separate package

### PR DESCRIPTION
**Pull request by ylongca**
- This is to prevent the test dependencies being downloaded by downstream dependencies.

### CODE REVIEW

File: ibclient_test/connector_test.go [hunk 1]
```diff
@@ -4,6 +4,7 @@ import (
 	"bytes"
```
Comment (pnagaraj80) [inline ibclient_test/connector_test.go:4]: This file is still part of ibclient package. Is this expected?

File: ibclient_test/objects_test.go [hunk 2]
```diff
@@ -3,27 +3,29 @@ package ibclient
 import (
```
Comment (pnagaraj80) [inline ibclient_test/objects_test.go:3]: This file is still part of ibclient package. Is this expected?

**Comment by daniel-garcia** [comment]
how does this affect unit test coverage reports?

**Comment by ylongca** [comment]
> how does this affect unit test coverage reports?

Unit test coverage cannot be reported because of this change.

**Review by hemanthKa677 — COMMENTED** [review]
Hi @ylongca, Is it really necessary to move test files to seperate package? Could you please confirm on this?
I am not sure , because dependenices will be anyway downloaded from go.mod file like ginkgo dependency for testing.

**Comment by daniel-garcia** [comment]
For now, lets create a branch that doesn't include the tests.

**Comment by ylongca** [comment]
> For now, lets create a branch that doesn't include the tests.

done.
https://github.com/infobloxopen/infoblox-go-client/tree/remove-unit-tests

**Pull request closed without merging**